package.json

```diff
@@ -82,19 +82,19 @@
     "@jest/transform": "^26.6.2",
     "@mdx-js/loader": "^2.0.0-rc.2",
     "@mdx-js/mdx": "^2.0.0-rc.2",
-    "@storybook/addon-a11y": "6.5.0-alpha.5",
-    "@storybook/addon-actions": "6.5.0-alpha.5",
-    "@storybook/addon-docs": "6.5.0-alpha.5",
-    "@storybook/addon-links": "6.5.0-alpha.5",
-    "@storybook/addon-storyshots": "6.5.0-alpha.5",
-    "@storybook/addons": "6.5.0-alpha.5",
-    "@storybook/builder-webpack5": "6.5.0-alpha.5",
-    "@storybook/manager-webpack5": "6.5.0-alpha.5",
+    "@storybook/addon-a11y": "6.5.0-alpha.18",
+    "@storybook/addon-actions": "6.5.0-alpha.18",
+    "@storybook/addon-docs": "6.5.0-alpha.18",
+    "@storybook/addon-links": "6.5.0-alpha.18",
+    "@storybook/addon-storyshots": "6.5.0-alpha.18",
+    "@storybook/addons": "6.5.0-alpha.18",
+    "@storybook/builder-webpack5": "6.5.0-alpha.18",
```

Comment:
we want to upgrade the storybook version to the latest and check how if it fix few storybook bugs

Reply:
do we have a list of bugs we'd like to get fixed?

Reply:
its mainly a bug with the order of the stories - the stories displayed by alphabet instead according to their load order even we did not defined it in the preview